quelpa.el

```diff
@@ -920,9 +941,12 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
     (quelpa-build--run-process-match
      "\\(.*\\)" dir "git" "rev-parse" "HEAD")))
 
-(defun quelpa-build--update-git-to-ref (dir ref)
-  "Update the git repo in DIR so that HEAD is REF."
-  (quelpa-build--run-process dir "git" "reset" "--hard" ref)
+(defun quelpa-build--update-git-to-ref (dir ref &optional force)
+  "Update the git repo in DIR so that HEAD is REF.
+This will perform a fast-forward merge or a reset if FORCE."
+  (if force
+      (quelpa-build--run-process dir "git" "reset" "--hard" ref)
+    (quelpa-build--run-process dir "git" "merge" "--ff-only" ref))
```

Comment:
Why merge instead of checkout?  In the PR comments you mention checkout, not merge.

Reply:
This is updated in latter commit, sorry for confusing situation like this